src/screens/TopicMemberScreen/index.js

```diff
@@ -9,9 +9,9 @@ import {getAllMemberTopic} from '../../service/topics';
 import dimen from '../../utils/dimen';
 import ShareUtils from '../../utils/share';
 import StringConstant from '../../utils/string/StringConstant';
-import UsersFragmentCommunity from '../DiscoveryScreenV2/fragment/UserFragmentCommunity';
 import NavHeader from '../TopicPageScreen/elements/NavHeader';
 import TopicMemberHeadline from './elements/TopicMemberHeadlineList';
+import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `UsersFragment` has been changed from `UsersFragmentCommunity`. If the new imported component has a different behavior or properties, it might affect the functionality of the `TopicMemberScreen`. Please ensure that this change is intended and the new component provides the same or improved functionality.
```diff
- import UsersFragmentCommunity from '../DiscoveryScreenV2/fragment/UserFragmentCommunity';
+ import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->